src/auro-calendar-cell.js

```diff
@@ -284,16 +300,16 @@ export class AuroCalendarCell extends LitElement {
         this.removeAttribute('renderForDateSlot');
       }
 
-      const popover = this.shadowRoot.querySelector('auro-popover');
+      const popover = this.shadowRoot.querySelector(this.popoverTag._$litStatic$);
```

Comment:
Can't we just query selector off the [auro-popover] attribute that should be applied for the custom named popover?

Reply:
I see what you are saying. Weirdly enough, when I change the selector from this to selecting the component by the attribute, the popover does not reposition as it gets closer to its boundaries. Not sure why, but not going to make this change right now.